apps/helpthechickens.com/src/app/page.tsx

```diff
@@ -3,18 +3,20 @@
 import { useScrollToId } from "@dxe/petitions-components/use-scroll-to-id";
 import { Button } from "@dxe/petitions-components/button";
 import { EmailPetition } from "@dxe/email-petition/email-petition";
-import { Dialog, DialogContent, DialogTrigger, DialogTitle } from "@dxe/petitions-components/dialog";
-import { PlayIcon } from "@/svg/play-icon";
 import { Section } from "@dxe/petitions-components/section";
-import Image from "next/image";
-import { VisuallyHidden } from "@radix-ui/react-visually-hidden";
 import { DEFAULT_MESSAGE } from "@/data/petition-message";
+import { useEffect, useState } from "react";
 
 export default function HomePage() {
   return (
     <div className="flex flex-col gap-6 items-center">
       <Hero />
       <PetitionSection />
+      <MoreBackgroundSection />
+      <KeyFindingsSection />
+      <FullInvestigatoryReport></FullInvestigatoryReport>
+      <PressHits></PressHits>
+      <MarketingVsReality></MarketingVsReality>
```

Comment:
nit: i usually use prefer to use self-closing tags if there are no children

Reply:
fixed